api/src/main/java/org/datacleaner/descriptors/DescriptorProvider.java

```diff
@@ -46,12 +46,6 @@
 public interface DescriptorProvider {
 
     /**
-     * @return a collection of {@link DescriptorProviderStatus} describing
-     *         current status.
-     */
-    public Collection<DescriptorProviderStatus> getStatus();
-
-    /**
```

Comment:
Okay, let's take it out. It's not so nice, but you're right that that the chance anyone us using it is _very_ slim. Then let's delete `DescriptorProviderStatus` as well, then.